api/client/formatter/custom.go

```diff
@@ -142,6 +143,20 @@ func (c *containerContext) Label(name string) string {
 	return c.c.Labels[name]
 }
 
+func (c *containerContext) Mounts() string {
```

Comment:
as a Capitalize Exported function, doesn't this need a comment? Shouldn't one of the validate passes catch this? Am I mistaken?